src/app/components/LoginForm.jsx

```diff
@@ -3,29 +3,53 @@
 import React, { useState } from 'react'
 /* eslint-enable no-unused-vars */
 import Link from 'next/link'
+import axios from 'axios'
+import { useRouter } from 'next/navigation'
 
 function LoginForm() {
 	const [showPassword, setShowPassword] = useState(false)
 
 	const togglePassword = () => {
 		setShowPassword(!showPassword)
 	}
+	const router = useRouter()
+
+	const isMobile = () => {
+		return typeof window !== 'undefined' ? window.innerWidth <= 768 : false
+	}
+
+	async function onSubmit(event) {
+		event.preventDefault()
+		const formData = new FormData(event.target)
+
+		axios
+			.post(process.env.NEXT_PUBLIC_BASE_URL + '/shared/auth/login', formData)
+			.then(function (response) {
+				document.cookie = `access_token=${response.data.access_token}; Secure; HttpOnly; SameSite=Strict`
+				document.cookie = `refresh_token=${response.data.refresh_token}; Secure; HttpOnly; SameSite=Strict`
+				const stateSidebar = isMobile() ? 'false' : 'true'
+				router.push(`/beneficiaries?showSidebar=${stateSidebar}`)
+			})
+			.catch(function (error) {
+				alert('Error al iniciar sesión: ' + error.response.data.detail)
```

Comment:
Error if you log in with a user that does not exist:
![image](https://github.com/ISPP-07/frontend-acat/assets/72880234/8a5cac9d-0332-4f9e-bc42-06fff0246b09)